--- translate-toolkit-0.10.1/translate/misc/xmlwrapper.py.python25	2007-01-09 15:00:41.000000000 +0330
+++ translate-toolkit-0.10.1/translate/misc/xmlwrapper.py	2007-01-09 15:01:13.000000000 +0330
@@ -21,7 +21,7 @@
 
 """simpler wrapper to the elementtree XML parser"""
 
-from elementtree import ElementTree
+from xml.etree import ElementTree
 # this is needed to prevent expat-version conflicts with wx >= 2.5.2.2
 from xml.parsers import expat
 
