--- translate-toolkit-1.0.1/translate/misc/xmlwrapper.py.python25	2007-12-19 14:30:40.000000000 +0200
+++ translate-toolkit-1.0.1/translate/misc/xmlwrapper.py	2007-12-19 14:31:15.000000000 +0200
@@ -21,7 +21,10 @@
 
 """simpler wrapper to the elementtree XML parser"""
 
-from elementtree import ElementTree
+try:
+  from xml.etree import ElementTree
+except ImportError:
+  from elementtree import ElementTree
 # this is needed to prevent expat-version conflicts with wx >= 2.5.2.2
 from xml.parsers import expat
 
